internal/aws/cwlogs/pusher.go

```diff
@@ -47,37 +47,37 @@ var (
 )
 
 // logEvent struct to present a log event.
-type logEvent struct {
-	inputLogEvent *cloudwatchlogs.InputLogEvent
+type LogEvent struct {
+	InputLogEvent *cloudwatchlogs.InputLogEvent
 	// The time which log generated.
-	logGeneratedTime time.Time
+	LogGeneratedTime time.Time
```

Comment:
Since there is no other time field here and we know we're dealing with logs, would this be better as `GeneratedTime`?